pkg/queue/multischedulingqueue/multi_scheduling_queue.go

```diff
@@ -52,52 +48,76 @@ func NewMultiSchedulingQueue(fw framework.Framework, podInitialBackoffSeconds in
 		podMaxBackoffSeconds: podMaxBackoffSeconds,
 	}
 
-	// TODO support mutil queue
+	// support mutil queue, delete default queue
```

Comment:
L51 is fine. But L52 contains non-English comment.

Reply:
@alwaysdark  
Can you remove the unused comments?

Reply:
ok